File: Personal/Currency.c
#include <stdio.h>
#include <curl/curl.h>

int main()
{

    CURL *curl;
    CURLcode res;
    curl = curl_easy_init();
    if (curl)
    {
        curl_easy_setopt(curl, CURLOPT_CUSTOMREQUEST, "GET");
        curl_easy_setopt(curl, CURLOPT_URL, "https://api.apilayer.com/exchangerates_data/convert?to=HKD&from=GBP&amount=10000");
        curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
        curl_easy_setopt(curl, CURLOPT_DEFAULT_PROTOCOL, "https");
        struct curl_slist *headers = NULL;
        headers = curl_slist_append(headers, "apikey: tiyERJ7OAXNhB6KSVg2mm7POiWr73S7J");
        curl_easy_setopt(curl, CURLOPT_HTTPHEADER, headers);

        res = curl_easy_perform(curl);
    }
    curl_easy_cleanup(curl);
}

File: Personal/Currency.py
import requests
import time

url = "https://api.apilayer.com/exchangerates_data/convert?to=HKD&from=GBP&amount=10000"

payload = {}
headers= {
  "apikey": "tiyERJ7OAXNhB6KSVg2mm7POiWr73S7J"
}

response = requests.request("GET", url, headers=headers, data = payload)

status_code = response.status_code
result = response.text

o = 0

while o == 0:
    print(result)
    time.sleep(1)